packages/syft/src/syft/core/node/common/action/run_class_method_action.py

```diff
@@ -155,7 +155,23 @@ def execute_action(self, node: AbstractNode, verify_key: VerifyKey) -> None:
 
             method_name = self.path.split(".")[-1]
 
-            if (
+            if "ShareTensor" in self.path:
```

Comment:
let's put a TODO comment here in the code to route based on something that partial string search.